Docs v.0.16 update: FS Browser & Cluster nodes Monitor (#810)
epam/cloud-pipeline: docs/md/release_notes/v.0.16/v.0.16_-_Release_notes.md

- [Extended recursive symlinks handling](#extended-recursive-symlinks-handling)
- [Displaying of the latest commit date/time](#displaying-of-the-latest-commit-datetime)
- [Renaming of the GitLab repository in case of Pipeline renaming](#renaming-of-the-gitlab-repository-in-case-of-pipeline-renaming)

> - [Allowing to expose compute node FS to upload and download files](#allowing-to-expose-compute-node-FS-to-upload-and-download-files)

Reviewer: For correct work of anchor-links in `Markdown` documents - link addresses should be written in the lowercase
```suggestion
- [Allowing to expose compute node FS to upload and download files](#allowing-to-expose-compute-node-fs-to-upload-and-download-files)
```

For more details see [here](../../manual/06_Manage_Pipeline/6.1._Create_and_configure_pipeline.md#edit-a-pipeline-info).

## Allowing to expose compute node FS to upload and download files

For the `interactive runs` users are processing data in ad-hoc manner, which requires upload data from the local storage to the cloud and download results from the cloud to the local storage.
Cloud Pipeline supports a number of options for the user to perform that data transfers for the interactive runs:

- via the Object Storage (using Web GUI or CLI)
- via the File Storage (using Web GUI, CLI, WebDav-mounted-drive)

Previously, view, download, upload and delete operations required an intermediate location (bucket or fs) to be used. It might confuse user when a small dataset shall be loaded to the specific location within a run's filesystem.

> In the current version, direct exposure of the run's filesystem is supported. The "Browse" hyperlink is displayed in the **Active Run** dashboard after a job had been initialized:         

Reviewer: ```suggestion
In the current version, direct exposure of the run's filesystem is supported. The **BROWSE** hyperlink is displayed on the **Run logs** page after a job had been initialized:  
```

    ![CP_v.0.16_ReleaseNotes](attachments/RN016_BrowseFilesystem_1.png)

> User can click the link and a Storage browser Web GUI will be loaded:

Reviewer: ```suggestion
User can click the link and a Storage browser Web GUI will be loaded:  
```

    ![CP_v.0.16_ReleaseNotes](attachments/RN016_BrowseFilesystem_2.png)

User is able to:

> - view files and directories

Reviewer: Add an empty line before the list
```suggestion

- view files and directories
```

> - download and delete files and directories

Reviewer: ```suggestion
- download and delete files and directories  
```

    ![CP_v.0.16_ReleaseNotes](attachments/RN016_BrowseFilesystem_3.png)

> - upload files and directories          

Reviewer: ```suggestion
- upload files and directories  
```

    ![CP_v.0.16_ReleaseNotes](attachments/RN016_BrowseFilesystem_4.png)

> - search files and directories

Reviewer: ```suggestion
- search files and directories  
```

    ![CP_v.0.16_ReleaseNotes](attachments/RN016_BrowseFilesystem_5.png)

> For more details see [here](../..\manual\15_Interactive_services\15.4._Expose_node_filesystem.md).

Reviewer: ```suggestion
For more details see [here](../../manual/15_Interactive_services/15.4._Expose_node_filesystem.md).
```

> 

Reviewer: Extra empty line
```suggestion
```
